pkgs/http/lib/src/base_response.dart

```diff
@@ -46,6 +46,19 @@ abstract class BaseResponse {
   /// If a header value contains whitespace then that whitespace may be replaced
   /// by a single space. Leading and trailing whitespace in header values are
   /// always removed.
+  ///
+  /// Please keep in mind that the Dart HTTP client is likely to be running side
```

Comment:
I think that this might be too verbose. How about:

Some headers may be excluded by the client for security or privacy reasons. For example, browser-cased clients can only return headers in the CORS safelist or specifically allowed by the server.